staging/src/k8s.io/legacy-cloud-providers/aws/aws.go

```diff
@@ -4347,6 +4354,11 @@ func (c *Cloud) EnsureLoadBalancerDeleted(ctx context.Context, clusterName strin
 
 		// Collect the security groups to delete
 		securityGroupIDs := map[string]struct{}{}
+		annotatedSgSet := map[string]bool{}
+		annotatedSgsList := getSGListFromAnnotation(service.Annotations[ServiceAnnotationLoadBalancerSecurityGroups])
```

Comment:
how about ServiceAnnotationLoadBalancerExtraSecurityGroups :D

Reply:
Yes Sure can be and should be added.